### PR TITLE
Add missing i18n tags and rerun the string extractor

### LINKED ISSUE #1094: Some user interface strings do not have i18n
### Problem

Some strings in the UI are missing a corresponding `i18n` tag or `L()` function call, which makes them stay ignored by the i18n system.

### Solution

Find such occurences and fix them.

### PR DESCRIPTION
**Pull request by snowteamer**
Closes #1094

### CODE REVIEW

File: frontend/views/pages/Home.vue [hunk 1]
```diff
@@ -31,8 +31,10 @@ main.c-splash(data-test='homeLogo' v-if='!$store.state.currentGroupId')
   .create-or-join(v-else key='body-create')
     .card
       svg-create-group
-      h3.is-title-3 Create
-      p Create a new group and invite your friends.
+      h3.is-title-3
+        i18n Create
+      p
+        i18n Create a new group and invite your friends.
```
Comment (taoeffect) [inline frontend/views/pages/Home.vue]: For both of these it's preferable to keep the line count small by using `i18n.is-title-3(tag='h3')` and `i18n(tag='p')` respectively

File: strings/french.strings [hunk 2]
```diff
@@ -426,6 +426,12 @@
 /* MISSING TRANSLATION - views/pages/GroupChat.vue */
 "{numMembers} members" = "{numMembers} members";
 
+/* MISSING TRANSLATION - views/pages/Home.vue */
+"Create a new group and invite your friends." = "Create a new group and invite your friends.";
+
+/* MISSING TRANSLATION - views/pages/Home.vue */
+"Create" = "Create";
+
```
Comment (taoeffect) [inline strings/french.strings:434]: Did you want to add translations to these?

**Review by taoeffect — CHANGES_REQUESTED** [review]
Minor issues, thanks for PR!

File: frontend/views/pages/Home.vue (hunk 1 above)
Comment (snowteamer) [reply frontend/views/pages/Home.vue]: Ok, I tought less tokens was better, but the difference is small anyway

File: strings/french.strings (hunk 2 above)
Comment (snowteamer) [reply strings/french.strings:434]: Sure, I am preparing another PR for these and other missing translations

**Review by taoeffect — APPROVED** [review]
Nice job! Approved!

**Pull request merged**